lib/ansible/plugins/callback/json.py

```diff
@@ -101,14 +101,22 @@ def v2_playbook_on_stats(self, stats):
             summary[h] = s
 
         custom_stats = {}
+        custom_stats_global = {}
+
         if self.get_option('show_custom_stats') and stats.custom:
-            custom_stats.update(dict((self._convert_host_to_name(k), v) for k, v in stats.custom.items()))
-            custom_stats.pop('_run', None)
+
+            for k in sorted(stats.custom.keys()):
```

Comment:
Is the `sorted()` actually useful here? You're adding the keys to a standard python dict, which is unsorted by nature.

Reply:
This was pre pr code i think, so the output of the callback plugin give a ansible_host sorted list.

should i remove it?

Reply:
thanks for the review, i removed it you where right.